services/mastodon/mastodon-follow.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService, NotFound } = require('..')
+const { metric } = require('../text-formatters')
+const { optionalUrl } = require('../validators')
+
+const schema = Joi.object({
+  username: Joi.string().required(),
+  followers_count: Joi.number().required(),
```

Comment:
Am I correct in assuming that this will always be a non-negative integer value? If so we should reuse that respective helper validator